src/main.rs

```diff
@@ -0,0 +1,182 @@
+use typedb_client::{
+    concept::{Attribute, Concept, Value},
+    Connection, DatabaseManager, Session,
+    SessionType::{Data, Schema},
+    TransactionType::{Read, Write}
+};
+use futures::{StreamExt};
+use  std::fs;
+use text_io::read;
+
+const TEST_DATABASE: &str = "codeforces-data-model";
+ 
+fn new_core_connection() -> typedb_client::Result<Connection> {
+    Connection::new_plaintext("localhost:1729")
+}
+
+async fn load_data(connection: Connection)->std::io::Result<()>{
+    let data=fs::read_to_string("./src/data.tql")?;    
+    let databases = DatabaseManager::new(connection.clone());
+    let session = Session::new(databases.get(TEST_DATABASE).await.unwrap(), Data).await.unwrap();
+    let transaction = session.transaction(Write).await.unwrap();
+    let _ = transaction.query().insert(data.as_str());
+    transaction.commit().await.unwrap();
+    println!("\nData Loaded Successfully\n");
+    Ok(())
+}
+
+async fn load_schema(connection: Connection)->std::io::Result<()>{
+    let schema = fs::read_to_string("./src/schema.tql")?;
+    // query_options(con.clone()).await.unwrap();
+    let databases = DatabaseManager::new(connection.clone());
+    if databases.contains(TEST_DATABASE).await.unwrap()==false {
+        let _ = databases.create(TEST_DATABASE).await;
+        // define schema
+        let session = Session::new(databases.get(TEST_DATABASE).await.unwrap(), Schema).await.unwrap();
+        let transaction = session.transaction(Write).await.unwrap();
+        transaction.query().define(schema.as_str()).await.unwrap();
+        transaction.commit().await.unwrap();
+        session.force_close().unwrap();
```

Comment:
You shouldn't need to call `.force_close()` here. Sessions should close themselves once dropped.